### DISCUSSION TITLE
Black Hole 2ch or 16ch Crash’s after approximately 60 Minutes of usage

### DISCUSSION THREAD

**Discussion by doniro007** [ExistentialAudio/BlackHole#342] [Q&A]
Macbook Air M1 11.2.3

I use Zoom and streamed the Audio over a Mulit-output-device to OBS to record it.
After round  60 Minutes it stops and I have to disconnect for Zoom and when I reconnect the output in Zoom it works again.

**Comment by devinroth** [comment]
Are you on the latest version?

This was a known issue that I’m sure I fixed with the latest version.

Devin Roth

**Reply by doniro007** [reply]
I just updated it yesterday uninstalled it and deleted all the files.

**Comment by devinroth** [comment]
Are you sure it’s a crash?

Maybe you just don’t have drift correction enabled.

Devin Roth

**Reply by doniro007** [reply]
no it wasn´t, thank´s  I´ll give it a try
